WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -571,4 +571,79 @@
       end
     end
   end
+
+  describe 'GET #my_competitions' do
+    let(:delegate1) { FactoryGirl.create(:delegate) }
+    let!(:organizer1) { FactoryGirl.create(:user) }
+    let!(:organizer2) { FactoryGirl.create(:user) }
+    let!(:future_competition1) { FactoryGirl.create(:competition, :registration_open, starts: 3.week.from_now, organizers: [organizer1], delegates: [delegate1], eventSpecs: "222 333") }
+    let!(:future_competition2) { FactoryGirl.create(:competition, :registration_open, starts: 2.weeks.from_now, organizers: [organizer1], eventSpecs: "222 333") }
+    let!(:future_competition3) { FactoryGirl.create(:competition, :registration_open, starts: 1.weeks.from_now, organizers: [organizer1], eventSpecs: "222 333") }
+    let!(:past_competition4) { FactoryGirl.create(:competition, :registration_open, starts: 1.month.ago, organizers: [organizer1], eventSpecs: "222 333") }
+    let!(:past_competition5) { FactoryGirl.create(:competition, :registration_open, starts: 2.month.ago, organizers: [organizer2], delegates: [delegate1], eventSpecs: "222 333") }
+    let!(:past_competition6) { FactoryGirl.create(:competition, :registration_open, starts: 3.month.ago, organizers: [organizer2], delegates: [delegate1], eventSpecs: "222 333") }
```

Comment:
When I updated the name to past_competition I thought to myself I should change the numbers too, but then I was too lazy :)